File: app/src/main/java/com/uralsiberianworks/neuralpushkin/ConversationRoom/HolderMessageView.java
package com.uralsiberianworks.neuralpushkin.ConversationRoom;


import android.view.View;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

import com.uralsiberianworks.neuralpushkin.R;




public class HolderMessageView extends RecyclerView.ViewHolder {

    private final TextView chatText;


    public HolderMessageView(View v) {
        super(v);
        chatText = (TextView) v.findViewById(R.id.tv_chat_text);
    }

    public TextView getChatText() {
        return chatText;
    }
}

File: app/src/main/java/com/uralsiberianworks/neuralpushkin/conversationRoom/HolderMessageView.kt
package com.uralsiberianworks.neuralpushkin.conversationRoom

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.uralsiberianworks.neuralpushkin.R

class HolderMessageView(v: View) : RecyclerView.ViewHolder(v) {
    val chatText: TextView

    init {
        chatText = v.findViewById<View>(R.id.tv_chat_text) as TextView
    }
}